hack/ci-deploy-release.sh

```diff
@@ -73,13 +73,13 @@ kubectl -n=scylla-operator rollout status --timeout=5m deployment.apps/scylla-op
 kubectl -n=scylla-operator rollout status --timeout=5m deployment.apps/webhook-server
 
 if [[ -z "${SO_NODECONFIG_PATH:-}" ]]; then
- echo "Skipping NodeConfig creation"
+ echo "Skipping NodeConfig and CSI driver creation"
 else
  kubectl_create -f="${SO_NODECONFIG_PATH}"
-fi
```

Comment:
I don't think SO_NODECONFIG_PATH should affect what happens with local-csi-driver. Please set up a dedicated var.

Reply:
Should it allow for overriding the local-csi-driver path though (so `SO_CSI_DRIVER_PATH` like in the regular deploy script)? Or just for skipping it?

Reply:
`SO_CSI_DRIVER_PATH` defaulting to the URL seems reasonable

Reply:
done